deeptime/numeric/__init__.py

```diff
@@ -36,3 +36,5 @@
 from ._eigen import eigs, eig_corr, sort_eigs, spd_eig, spd_truncated_svd, \
     spd_inv, spd_inv_split, spd_inv_sqrt, ZeroRankError
 from ._norm import schatten_norm
+
+from ._numeric_bindings import kdot, ksum, logsumexp_pair #, logsumexp_kahan_inplace, logsumexp_sort_kahan_inplace
```

Comment:
what about the inplace versions? Also the methods should appear in the module documentation

Reply:
They are not referenced by any python code, only by the cpp code. Keep them in or remove?

Reply:
remove